SpotifyApp/SpotifyApp/SpotifyAlbums.json

```diff
@@ -0,0 +1,1711 @@
+{
```

Comment:
Thank you, that was the problem I was having. Had a tough time understanding JSON notation but I deal with it better now.